rest/admin_api.go

```diff
@@ -707,6 +709,7 @@ func (h *handler) getUserInfo() error {
 	}
 
 	bytes, err := marshalPrincipal(user)
+	h.setHeader("Content-Type", "application/json")
```

Comment:
We could remove the `h.response.Write` below, and use `h.writeRawJSON(bytes)` here instead to avoid manually setting the header.

Same for `getUsers()` and `getRoles()`.